Languages/IronPython/IronPython/Modules/sys.cs

```diff
@@ -177,7 +177,7 @@ public static class SysModule {
 
         public static object getfilesystemencoding() {
             if(Environment.OSVersion.Platform == PlatformID.Unix)
-                return "utf-8";
+                return "UTF-8";
```

Comment:
Why the change to uppercase? In https://docs.python.org/2/library/sys.html#sys.getfilesystemencoding it appears to be lowercase.